tests/test_router.py

```diff
@@ -6,48 +6,39 @@
 
 
 class TestRouter:
-    def setup_method(self) -> None:
-        # pylint: disable= attribute-defined-outside-init
-        self.router = Router()
-        self.router.add_route("/", "GET", "x")
+    def test___init__(self, sample_router: Router) -> None:
```

Comment:
I would add TODO somewhere here that we should improve the router tests.

e.g. this one should not exist 😉